doc/madman/madadv.help

```diff
@@ -0,0 +1,331 @@
+ZORK User Information
+
+I. Getting a ZORK
+ 
+    a. ITS:  Say ZORK^K to DDT rather than :ZORK to get a zork. 
```

Comment:
Any clue what the difference between ^K and a colon command would be?  When I looked in DDT, I didn't see a difference.

Reply:
If you already have a ZORK job, ZORK^K will get rid of it before starting a new one. I guess they were worried about people accidentally running multiple copies.